test/javascripts/acceptance/timezone-offset-test.js

```diff
@@ -0,0 +1,358 @@
+import { acceptance } from "discourse/tests/helpers/qunit-helpers";
+import { visit } from "@ember/test-helpers";
+import { test } from "qunit";
+
+const topicResponse = {
+  post_stream: {
+    posts: [
+      {
+        id: 375,
+        name: null,
+        username: "jan",
+        avatar_template: "/letter_avatar_proxy/v4/letter/j/ce7236/{size}.png",
+        created_at: "2023-09-08T16:50:07.638Z",
+        cooked:
```

Comment:
Cool, good to see an acceptance test here! 

Maybe add the `raw: ` value as well? It's not going to affect the test, but makes this a bit more readable, i.e. this raw markdown will generate that markup and calendar.